pkg/core/region.go

```diff
@@ -718,19 +724,16 @@ func GenerateRegionGuideFunc(enableLog bool) RegionGuideFunc {
 	}
 	// Save to storage if meta is updated.
 	// Save to cache if meta or leader is updated, or contains any down/pending peer.
-	// Mark isNew if the region in cache does not have leader.
-	return func(region, origin *RegionInfo) (isNew, saveKV, saveCache, needSync bool) {
+	// Now, no need to hold `isNew` because `saveCache` must be true if the region in cache does not have leader.
```

Comment:
maybe can move this statement to pr description
```suggestion
```

Reply:
Got it. But I think we can keep this comment in code.

Reply:
Oh 😯 I mean, after your code merged, `isNew` is a word which doesn't exist and could be confused others.

Reply:
OK